src/Upgrader/Tasks/UpgraderTask.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Tasks;
+
+use Codebase\Infrastructure\Service\CodebaseService;
+use CodeCompliance\Application\Service\CodeComplianceServiceInterface;
+use SprykerSdk\SdkContracts\Entity\Lifecycle\LifecycleInterface;
+use SprykerSdk\SdkContracts\Entity\TaskInterface;
+use Upgrade\Application\Service\UpgradeServiceInterface;
+use Upgrader\Commands\Evaluate\Analyze\AnalyzeCommand;
+use Upgrader\Commands\Evaluate\Report\ReportCommand;
+use Upgrader\Commands\Upgrade\UpgradeCommand;
+use Upgrader\Configuration\ConfigurationProvider;
+use Upgrader\Lifecycle\Lifecycle;
+
+class UpgraderTask implements TaskInterface
```

Comment:
Does it work properly?
Does each commands executes one by one or SDK doesn't support this approach?
Please remove this command if no.

Reply:
You are right 
SDK support only approach: one task -> one command 

Deleted UpgraderTask